concrete/src/Entity/Express/Entity.php

```diff
@@ -221,6 +221,22 @@ public function setDescription($description)
     }
 
     /**
+     * @param string $format
+     * @return string
+     */
+    public function getEntityDisplayDescription($format = 'html')
+    {
+        $value = t($this->getDescription());
```

Comment:
The problem is that entities are not meant to support multilanguage (at least for now)